cvat/apps/limit_manager/serializers.py

```diff
@@ -0,0 +1,60 @@
+from rest_framework.serializers import ModelSerializer, SlugRelatedField
+
+from django.conf import settings
+from cvat.apps.engine.models import User
+from cvat.apps.engine.serializers import BasicUserSerializer
+from cvat.apps.organizations.serializers import OrganizationReadSerializer
+from cvat.apps.organizations.models import Organization
+
+from .models import Limitation
+
+org_limitation_fields = tuple(settings.ORG_LIMITS_MAPPING.values())
+user_limitation_fields = tuple(settings.USER_LIMITS_MAPPING.values())
+
+
+class UserLimitationReadSerializer(ModelSerializer):
```

Comment:
I believe we should have 1 serialize and 2 tables in DB. One for users and another one for organizations.